systemtests/tests/bconsole-status-client/testrunner

```diff
@@ -6,13 +6,15 @@
 TestName="$(basename "$(pwd)")"
 export TestName
 
-JobName=bconsole-status-client
+#shellcheck source=../environment.in
 . ./environment
+
+JobName=backup-bareos-fd
```

Comment:
is this correct? The original JobName was "bconsole-status-client"

Reply:
JobName is irrelevant here as  no job is run at all in this test.